internal/commands/vaultsecrets/integrations/displayer.go

```diff
@@ -44,15 +44,22 @@ func (t *twilioDisplayer) previewTwilioIntegrationsPayload() any {
 }
 
 func (t *twilioDisplayer) FieldTemplates() []format.Field {
-	return []format.Field{
+	fields := []format.Field{
 		{
 			Name:        "Integration Name",
 			ValueFormat: "{{ .IntegrationName }}",
 		},
-		{
-			Name:        "Account SID",
-			ValueFormat: "{{ .TwilioAccountSid }}",
-		},
+	}
+
+	if t.single {
+		return append(fields, []format.Field{
```

Comment:
curious, why does this go in only if a single integration is being shown? Shouldn't `TwilioAccountSid` be in all the response entries?

Reply:
We currently don't populate the credential details on LIST requests (we may add them later but there were some performance concerns with queries to tokenization)